sources/custom controls/PlayerViewController.swift

```diff
@@ -69,6 +82,18 @@ public final class PlayerViewController: UIViewController {
             adControls.didMove(toParent: self)
         }
         
+        /* Create -> configure -> add loading indicator to video view */ do {
+            let loadingImage = UIImage(named: "icon-loading", in: Bundle(for: AdVideoControls.self), compatibleWith: view.traitCollection)
+            loadingImageView = UIImageView(image: loadingImage)
+            loadingImageView.center = view.center
+            loadingImageView.autoresizingMask = [.flexibleLeftMargin,
```

Comment:
Why autoresizing masks? 
Let's use auto layouts

Reply:
And maybe make sense to move this loading view into separate view, with two public methods start/stop loading or `show/hideLoadingView(on superView: UIView)`.

Reply:
Not a problem. And those animation methods can be added right for this view